sourcemash/frontend/static/logotext.svg

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe this will be helpful? https://shkspr.mobi/blog/2013/03/inkscape-cropping-svg-files-on-the-command-line/